content/en/docs/concepts/services-networking/service.md

```diff
@@ -1352,211 +1177,26 @@ spec:
 ```
 
-## Shortcomings
-
-Using the userspace proxy for VIPs works at small to medium scale, but will
-not scale to very large clusters with thousands of Services.  The
-[original design proposal for portals](https://github.com/kubernetes/kubernetes/issues/1107)
-has more details on this.
-
-Using the userspace proxy obscures the source IP address of a packet accessing
-a Service.
-This makes some kinds of network filtering (firewalling) impossible.  The iptables
-proxy mode does not
-obscure in-cluster source IPs, but it does still impact clients coming through
-a load balancer or node-port.
-
-The `Type` field is designed as nested functionality - each level adds to the
-previous.  This is not strictly required on all cloud providers (e.g. Google Compute Engine does
-not need to allocate a `NodePort` to make `LoadBalancer` work, but AWS does)
-but the Kubernetes API design for Service requires it anyway.
-
-## Virtual IP implementation {#the-gory-details-of-virtual-ips}
-
-The previous information should be sufficient for many people who want to
-use Services.  However, there is a lot going on behind the scenes that may be
-worth understanding.
-
-### Avoiding collisions
-
-One of the primary philosophies of Kubernetes is that you should not be
-exposed to situations that could cause your actions to fail through no fault
-of your own. For the design of the Service resource, this means not making
-you choose your own port number if that choice might collide with
-someone else's choice.  That is an isolation failure.
-
-In order to allow you to choose a port number for your Services, we must
-ensure that no two Services can collide. Kubernetes does that by allocating each
-Service its own IP address from within the `service-cluster-ip-range`
-CIDR range that is configured for the API server.
-
-To ensure each Service receives a unique IP, an internal allocator atomically
-updates a global allocation map in {{< glossary_tooltip term_id="etcd" >}}
-prior to creating each Service. The map object must exist in the registry for
-Services to get IP address assignments, otherwise creations will
-fail with a message indicating an IP address could not be allocated.
-
-In the control plane, a background controller is responsible for creating that
-map (needed to support migrating from older versions of Kubernetes that used
-in-memory locking). Kubernetes also uses controllers to check for invalid
-assignments (e.g. due to administrator intervention) and for cleaning up allocated
-IP addresses that are no longer used by any Services.
-
-#### IP address ranges for `type: ClusterIP` Services {#service-ip-static-sub-range}
-
-{{< feature-state for_k8s_version="v1.25" state="beta" >}}
-However, there is a problem with this `ClusterIP` allocation strategy, because a user
-can also [choose their own address for the service](#choosing-your-own-ip-address).
-This could result in a conflict if the internal allocator selects the same IP address
-for another Service.
```

Comment:
This paragraph is deleted, right?

Reply:
Yes, this paragraph is deleted. The moved text does not follow the previous paragraph and I'm now omitting the motivation.

Given that `ServiceIPStaticSubrange` is enabled by default it doesn't feel particularly important to explain the motivation about why you might want to enable it.